src/mumble/MainWindow.cpp

```diff
@@ -1739,6 +1751,13 @@ void MainWindow::qmListener_aboutToShow() {
 
 	if (self) {
 		qmListener->addAction(qaListenerLocalVolume);
+		Channel *channel = getContextMenuChannel();
+		ClientUser *user = getContextMenuUser();
```

Comment:
What is the `user` being used for?

Reply:
`ListenerDialog` stops the dialog from being created if `nullptr` is returned. I don't know if there's a reason for that but I didn't feel confident in changing it.

Reply:
I think I also just copy&pasted from the user dialog. Looking at this now it seems like the user pointer is never actually used. As such we don't have to check for it here either, I would presume :thinking: